test/unit3/test_strptime.py

```diff
@@ -83,8 +83,8 @@ def test_date_time(self):
         #  output.
         magic_date = (1999, 3, 17, 22, 44, 55, 2, 76, 0)
         strftime_output = time.strftime("%c", magic_date)
```

Comment:
On my local machine strftime_output is set to:
`Wed 17 Mar 1999 10:44:55 PM Australian Eastern Daylight Time`